src/modules/LaptopDetail/components/CommentTab/CommentCard.tsx

```diff
@@ -0,0 +1,117 @@
+import { Divider, Dropdown, MenuProps, Rate, Typography } from "antd";
+import { useTranslation } from "react-i18next";
+import moment from "moment";
+
+import { MenuDotsIcon, PencilIcon, TrashIcon } from "icons";
+import { useAppDispatch, useAppSelector } from "hooks/useRedux";
+import { RootState } from "redux/store";
+import {
+  deleteCommentByIdActionRequest,
+  setCommentItem
+} from "redux/features/laptop.slice";
+import { IComment } from "types/comment.model";
+import {
+  CommentItem,
+  CommentItemDescription,
+  UserCommentInfo
+} from "./CommentTab.styled";
+
+const { Text, Title } = Typography;
+
+interface CommentCardProps {
+  commentId: string;
+  userId: string;
+  avatar: string;
+  fullName: string;
+  comment: string;
+  rating: number;
+  updatedAt: string;
+}
+
+const CommentCard = ({
+  commentId,
+  userId,
+  avatar,
+  fullName,
+  comment,
+  rating,
+  updatedAt
+}: CommentCardProps) => {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+  const { userInfo } = useAppSelector((state: RootState) => state.auth);
+  const { commentList } = useAppSelector((state: RootState) => state.laptop);
+
+  const handleClickUpdateComment = (selectedCommentId: string) => {
```

Comment:
This function don't need use useMemo. Because this function only trigger when click button update comment. So, use useMemo in this circumtance is unnessary